deps/anvil/core_contract_deployment.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
This whole script is in Orchestrator, right? 
Probably we should take it from there and not copy paste (maybe they change it)
Add to improvement list :)

Reply:
It was taken from Orchestrator indeed. [This](https://github.com/madara-alliance/madara-orchestrator/blob/main/scripts/dummy_contract_deployment.sh) is the original script. The issue with this script is that it requires both L1 (Anvil) and Madara running at the same time because it interacts with both services.
Unfortunately, that is not possible, unless to spin up Madara with `--no-l1-sync`. Therefore, the script was split into 2: 
1) The part that interacts with Anvil runs first, then Madara can be spinned up.
2) The part that interacts with Madara. There are some outputs from the first one that should be included in the execution from the second script (so far, the values are hardcoded)